### PR TITLE
vfs: change the error code of ioctl for non-supported commands

### PR DESCRIPTION
**Pull request by SandyXSD**

### CODE REVIEW

File: pkg/vfs/vfs_unix.go [hunk 1]
```diff
@@ -305,7 +305,7 @@ func (v *VFS) Ioctl(ctx Context, ino Ino, cmd uint32, arg uint64, bufIn, bufOut
 	defer func() { logit(ctx, "ioctl (%d,0x%X,0x%X,%v,%v): %s", ino, cmd, arg, bufIn, bufOut, strerr(err)) }()
 	switch cmd {
 	default:
-		err = syscall.ENOTTY
+		err = syscall.ENOSYS
```
Comment (davies) [inline pkg/vfs/vfs_unix.go:308]: ENOSYS means it don't support all ioctl operations, ENOTSUP will be better?

Comment (SandyXSD) [reply pkg/vfs/vfs_unix.go:308]: Waiting for the reporter to see if `ENOTSUP` works on his case.

Comment (SandyXSD) [reply pkg/vfs/vfs_unix.go:308]: Neither `ENOSYS` nor `ENOTSUP` can solve reporter's issue.

Comment (davies) [reply pkg/vfs/vfs_unix.go:308]: Then we should use ENOTSUP, otherwise the first return of ENOSYS will disable this feature.

**Pull request closed without merging**